test/file1-1.tsx

```diff
@@ -0,0 +1,129 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
```

Comment:
The @jsxRuntime declaration is not necessary since it is set to classic, which is the default value.